organization/pre-workshop-survey.md

```diff
@@ -0,0 +1,84 @@
+# Pre-Workshop Survey
+
+1. I consent to this survey *
+
+    Yes
+
+1. Are you age 18 or above? (we should not collect if under 18)* 
+
+    Yes
+    No
+
+1. In a few words, what is your most important reason for attending this workshop?
+
+    
+
+1. Below are some reasons we think people will come to Toronto Mesh workshops. Some of these reasons may overlap with the reasons you suggested above. How important were these reasons to you?
+  	
+    _Not at all important 	Slightly important 	Important 	Very important 	N/A_  
+    Improve your knowledge of networking and the internet  
+    Join an active project or group of enthusiasts  
+    Experiment with new software and tools related to your interests  
+    Network with other workshop attendees   	
+    Learn new or update existing skills  	
+
+1. Besides the course content, what importance did the following factors have on your choice to attend the workshop?
+  	Not important at all 	Slightly important 	Important 	Very important 	Extremely important
+
+    Attending an in-person/live workshop  
+    Location and timing of workshop  
+    Learning with friends/colleagues  
+    Other (please specify)  
+
+6. How did you hear about Toronto Mesh / Building the Peer-to-Peer Internet workshop? (Select all that apply)
+
+    Friend/colleague  
+    Conference/meeting/seminar  
+    Library website or flyer  
+    Funding organization or program officer  
+    Journal or publication  
+    Our website  
+    Social Media (Twitter, Facebook, etc.)  
+    Other (please specify)  
+
+1. If you experienced any strong concerns about attending this workshop, can you let us know what they were? Did we address these concerns, or were there ways we could have addressed them? w
```

Comment:
This seems like a post-workshop question?

Reply:
I think I just failed to get the tense right, but I think a pre question on concerns is important